packages/web/components/Guild/GuildForm.tsx

```diff
@@ -293,6 +314,124 @@ export const GuildForm: React.FC<Props> = ({
             ))}
           </Select>
         </Field>
+        <Box
+          borderWidth="1px"
+          borderRadius="lg"
+          borderColor="rgba(255, 255, 255, 0.25)"
+          p={4}
+        >
+          <Text mb={2}>DAO information</Text>
+          <Text fontSize="sm" mb={4}>
+            If your guild has one or more DAOs, enter their information here. If
+            your DAO is in DAOHaus, we will look up its information from the{' '}
+            <MetaLink
+              isExternal
+              href="https://thegraph.com/hosted-service/subgraph/odyssy-automaton/daohaus"
+            >
+              DAOHaus Subgraph
+            </MetaLink>
+            . Otherwise you can provide your own URL to link to.
+          </Text>
+          {daoFields.map((_, index) => (
+            <Box
+              key={index}
+              background="rgba(255, 255, 255, 0.1)"
+              borderRadius="lg"
+              p={2}
+              mb={4}
+            >
+              {daoFields.length > 1 && (
+                <HStack justifyContent="space-between">
+                  <Text size="lg" ml={2}>
+                    {index + 1}.
+                  </Text>
+                  <CloseButton size="sm" onClick={() => remove(index)} />
+                </HStack>
+              )}
+              <Box borderRadius="lg" p={2}>
+                <Flex direction="row">
+                  <Box mr={4}>
+                    <Field label="Label">
+                      <Input
+                        placeholder="DAO, Multisig, etc"
+                        {...register(`daos.${index}.label`)}
+                        background="dark"
+                      />
+                    </Field>
+                  </Box>
+                  <Box flex="1">
+                    <Field label="URL">
+                      <Input
+                        placeholder="https://"
+                        {...register(`daos.${index}.url`)}
+                        background="dark"
+                      />
+                      <FieldDescription>
+                        An optional URL to link this address to
```

Comment:
We may want to make dao URL mandatory if this is not a DaoHaus dao. Perhaps a checkbox for the user to select if guild is DaoHaus dao or not?